template/src/ontology/Makefile.jinja2

```diff
@@ -396,7 +396,7 @@ $(IMPORTDIR)/%_import.owl: {% if project.import_group.use_base_merging %}$(MIRRO
 		query --update ../sparql/inject-subset-declaration.ru --update ../sparql/inject-synonymtype-declaration.ru --update ../sparql/postprocess-module.ru \
 		$(ANNOTATE_CONVERT_FILE); fi
 {% elif 'custom' == project.import_group.module_type %}
-$(IMPORTDIR)/%_import.owl: {% if project.import_group.use_base_merging %}$(MIRRORDIR)/merged.owl{% else %}$(MIRRORDIR)/%.owl{% endif %}
+$(IMPORTDIR)/%_import.owl: {% if project.import_group.use_base_merging %}$(MIRRORDIR)/merged.owl{% elif 'custom' == ont.mirror_type %}$(MIRRORDIR)/%.owl{% endif %}
```

Comment:
I dont like this.. ODK should never change the default behaviour! Even if its `custom` module type, the mirror can be the default normal. I think the condition here should be 

```
{% elif 'no_mirror' != ont.mirror_type %}
```

Reply:
Correct me if I am wrong though!

Reply:
You're right. This should be in sync with the other import part.